hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -1,131 +1,110 @@
+If `1` then enable UART debug output on GPIO 14 and 15. Configure the receiving debug terminal at 115200bps, 8 bits, no parity bits, 1 stop bit.
 
-Default: 0  
+Default: `0`  
 Version: All  
 
 ### WAKE_ON_GPIO
 
-If 1 then 'sudo halt' will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.
+If `1` then `sudo halt` will run in a lower power mode until either GPIO3 or GLOBAL_EN are shorted to ground.
 
-Default: 1 (0 in original version of bootloader 2019-05-10)  
+Default: `1` (`0` in original version of bootloader 2019-05-10)  
 Version: All  
 
 ### POWER_OFF_ON_HALT
 
-If 1 and WAKE_ON_GPIO=0 then switch off all PMIC outputs in halt. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
+If `1` and `WAKE_ON_GPIO=0` then `sudo halt` will switch off all PMIC outputs. This is lowest possible power state for halt but may cause problems with some HATs because 5V will still be on. GLOBAL_EN must be shorted to ground to boot.
 
-Default: 0  
+Default: `0`  
 Version: 2019-07-15  
 
 ### BOOT_ORDER
-The BOOT_ORDER setting allows flexible configuration for the priority of different bootmodes. It is represented as 32bit unsigned integer where each nibble represents a bootmode. The bootmodes are attempted in lowest significant nibble to highest significant nibble order.
-
-E.g. 0x21 means try SD first followed by network boot then stop. Whereas 0x2 would mean try network boot and then stop without trying to boot from the SD card.
+The `BOOT_ORDER` setting allows flexible configuration for the priority of different bootmodes. It is represented as 32bit unsigned integer where each nibble represents a bootmode. The bootmodes are attempted in lowest significant nibble to highest significant nibble order.
 
+E.g.`0xf41` means continuously trying SD card followed by USB mass storage. Whereas, `0xf12` means continuously trying network boot followed by SD card boot. 
 The retry counters are reset when switching to the next boot mode.
 
-BOOT_ORDER fields  
+`BOOT_ORDER` fields  
 The BOOT_ORDER property defines the sequence for the different boot modes. It is read right to left and up to 8 digits may be defined.
 
-* 0x0 - NONE (stop with error pattern)
-* 0x1 - SD CARD
-* 0x2 - NETWORK
-* 0x3 - USB device boot - Reserved - Compute Module only.
-* 0x4 - USB mass storage boot (since 2020-09-03)
-* 0xf - RESTART (loop) - start again with the first boot order field. (since 2020-09-03)
+| Value | Mode      | Description                                                                                       |
+|-------|-----------|---------------------------------------------------------------------------------------------------|
+|  0x1  |  SD CARD  |  SD card (or EMMC on Compute Module 4)                                                            |
```

Comment:
I'm wondering if `or` should be removed, since CM4 only has eMMC. It is also possible to use SD card via GPIO on the CM4, but I'm not sure if you can boot from it.

Reply:
Also `EMMC` -> `eMMC`

Reply:
CM4 has either eMMC or SD card on LITE. It's not currently possible to boot from SDHOST controller

Reply:
The CM4 Lite doesn't have eMMC, and requires the use of an external SD card.

Reply:
> The CM4 Lite doesn't have eMMC, and requires the use of an external SD card.

That's what I thought. So without working SD card boot on CM4, the word `or` should be removed.